ENIGMAsystem/SHELL/Platforms/Cocoa/CocoaMain.cpp

```diff
@@ -98,4 +99,19 @@ string parameter_string(int n) {
   return string("");
 }
 
+string shellscript_evaluate(string command, size_t buffer_size) {
```

Comment:
Apple is POSIX-compliant and should be able to use this function from a more general POSIX source.

Reply:
I was going to ask where you want the function but you stopped responding to me on hangouts so I just assumed make a pr and hope you'd respond after that

Reply:
Right next to `execute_shell` is probably the best place...